author/serializers.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers
+from author.models import Author
+
+
+class CreateAuthorSerializer(serializers.ModelSerializer):
+
+    class Meta:
+
+        model = Author
+        fields = ('id', 'name', 'surname', 'patronymic')
+
+
+class UpdateAuthorSerializer(serializers.ModelSerializer):
```

Comment:
UpdateAuthorSerializer is same as CreateAuthorSerializer as well as RetrieveAuthorsSeliarizer, I reccomend to make only one

Reply:
Fixed